libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts

```diff
@@ -304,10 +322,66 @@ export class TigerWorkspaceDashboards implements IWorkspaceDashboardsService {
         );
     };
 
-    public exportDashboardToPdf = async () => {
-        throw new NotSupported("Tiger backend does not support export to PDF.");
+    public exportDashboardToPdf = async (
+        dashboardRef: ObjRef,
+        filters?: FilterContextItem[],
+    ): Promise<string> => {
+        const dashboardId = await objRefToIdentifier(dashboardRef, this.authCall);
+
+        // skip all time date filter from stored filters, when missing, it's correctly
+        // restored to All time during the load later
+        const withoutAllTime = (filters || []).filter((f) => !isAllTimeDashboardDateFilter(f));
+
+        return this.authCall(async (client) => {
+            const dashboardResponse = await client.entities.getEntityAnalyticalDashboards(
+                {
+                    workspaceId: this.workspace,
+                    objectId: dashboardId,
+                },
+                {
+                    headers: jsonApiHeaders,
+                },
+            );
+
+            const { title } = convertDashboard(dashboardResponse.data);
+            const pdfExportRequest = {
+                fileName: title,
+                dashboardId,
+                metadata: convertToBackendExportMetadata({ filters: withoutAllTime }),
+            };
+            const pdfExport = await client.export.createPdfExport({
+                workspaceId: this.workspace,
+                pdfExportRequest,
+            });
+
+            return await this.handleExportResultPolling(client, {
+                workspaceId: this.workspace,
+                exportId: pdfExport?.data?.exportResult,
+            });
+        });
     };
 
+    private async handleExportResultPolling(
+        client: ITigerClient,
+        payload: { exportId: string; workspaceId: string },
+    ): Promise<string> {
+        for (let i = 0; i < MAX_POLL_ATTEMPTS; i++) {
+            const result = await client.export.getExportedFile(payload, {
+                transformResponse: (x) => x,
+            });
+
+            if (result?.status === 200) {
+                return result?.config?.url || "";
+            }
+
+            await new Promise((resolve) => setTimeout(resolve, DEFAULT_POLL_DELAY));
+        }
+
+        throw new Error(
```

Comment:
Added this way, what would you think, is it ok or should we aim for more general timeout error (i.e. TIMEOUT_ERROR instead of TIMEOUT_POLLING_ERROR) https://github.com/gooddata/gooddata-ui-sdk/commit/6f78923e93d97fb6a7e880267ae99a70bc6c44b9